sdk/storage/Azure.Storage.Files.DataLake/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0004: DO provide both asynchronous and synchronous variants for all service methods.", Justification = "")]
+[assembly: SuppressMessage("Usage", "AZC0002: Client method should have an optional CancellationToken (both name and it being optional matters) or a RequestContext as the last parameter.", Justification = "")]
```

Comment:
This is another instance where we are suppressing warnings that worked correctly before for this library.  We should have a strong justification for why we're doing this.  For this one, I'm also wondering what changed in AZC0002 that made us suppress it for this library?  There's no mention of it changing in either of the PR warnings, so am concerned we may have introduced a bug unnecessarily.